CHANGELOG.md

```diff
@@ -1,9 +1,30 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 but this project DOES NOT adhere to [Semantic Versioning](http://semver.org/).
 
+## 8.0
+
+* Upgrade to .NET 8
+* Add `LeanCode.Kratos`
+* Remove custom outbox in favor of MassTransit Outbox
+* Switch to `.Add*` pattern for DI configuration, removing Autofac and `IAppModule`s
+* Use Endpoint Routing for RemoteCQRS
+* Remove custom pipelines, use ASP.NET mechanism of middlewares
+* Remove `AppContext` in favor of using `HttpContext` directly
+* Support Azure Workload Identity
+* Improve CoreLib-provided logging, tracing and metrics
+* Remove singleton/scoped FluentValidation configuration, remove `RuleForAsync` in favor of `CustomAsync`
+* Introduce exception translation middleware,
+* Add Ulids as a valid source-generated ID
+* Switch to `System.TimeProvider`, removing `ITimeProvider`
+* Improved PostgreSQL handling
+* UserId is now a generic type in most of the helpers
+* `IIdentifiable` is not `IEntity`
```

Comment:
Let's be more straightforward with that one:
```suggestion
* Rename `IIdentifiable` to `IEntity`
```

Reply:
I could swear that this was a sentence in the original form. Will fix!